mintscript/src/tests/translate/snap.translate.tlock.transfer_03.txt

```diff
@@ -0,0 +1,8 @@
+WitnessScript(
+    Threshold(
+        Threshold {
+            required: 0,
+            conditions: [],
+        },
+    ),
+)
```

Comment:
All tlock transfer seem to have no signature verification

Reply:
That is correct. The timelock-only mode is intended to only have timelocks.